enjoliver-api/tests/unit/test_machine_discovery_repo.py

```diff
@@ -1,59 +1,63 @@
 import copy
 import unittest
 
-from app import smartdb, model
-from common import posts
-from model import MachineInterface, Machine, MachineDisk, Chassis, ChassisPort
-from repositories.machine_discovery_repo import DiscoveryRepository
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from enjoliver.db import session_commit
+from enjoliver.model import MachineInterface, Machine, MachineDisk, Chassis, ChassisPort, Base
+from enjoliver.repositories.machine_discovery import MachineDiscoveryRepository
+
+from tests.fixtures import posts
 
 
 class TestMachineStateRepo(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        db_uri = 'sqlite:///:memory:'
-
-        cls.smart = smartdb.SmartDatabaseClient(db_uri)
+        db_uri = 'postgresql+psycopg2://localhost/enjoliver_testing'
```

Comment:
Should we have a wrapper/test harness around those test to setup/launch a PostgreSQL db and tear it down after ?